databases/de-database-schema/src/main/tables/76_job_status_updates.sql

```diff
@@ -0,0 +1,48 @@
+SET search_path = public, pg_catalog;
+
+--
+-- Records job status updates
+--
+CREATE TABLE job_status_updates (
+    id uuid NOT NULL DEFAULT uuid_generate_v4(),
+
+    -- corresponds to the external_id field in the job_steps table
+    external_id character varying(64) NOT NULL,
```

Comment:
Is `external_id` always populated in the `job_steps` table? I thought it only corresponded to Agave app IDs?